experiments/download_ssl4eo.py

```diff
@@ -393,98 +288,111 @@ def update(self, delta: int = 1) -> int:
             return self.value
 
 
-def fix_random_seeds(seed: int = 42) -> None:
-    """
-    Fix random seeds.
-    """
-    np.random.seed(seed)
-
-
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
-    parser.add_argument("--save-path", type=str, default="./data/")  # dir to save data
-    parser.add_argument("--debug", action="store_true")
-    parser.add_argument("--cloud-pct", type=int, default=20)
-    parser.add_argument("--num-workers", type=int, default=8)
-    parser.add_argument("--log-freq", type=int, default=10)  # print frequency
     parser.add_argument(
-        "--resume", type=str, default=None
-    )  # resume from existing coordinates
+        "--save-path", type=str, default="./data/", help="dir to save data"
+    )
+    # collection properties
+    parser.add_argument(
+        "--collection", type=str, default="COPERNICUS/S2", help="GEE collection name"
+    )
+    parser.add_argument(
+        "--qa-band", type=str, default="QA60", help="qa band name (optional)"
+    )  # optional
+    parser.add_argument(
+        "--qa-cloud-bit", type=int, default=10, help="qa band cloud bit (optional)"
```

Comment:
Landsat has several cloud-related bits (https://developers.google.com/earth-engine/datasets/catalog/LANDSAT_LC08_C02_T1_TOA). Should we only consider bit 3, or other cloud bits too?

Reply:
This I don't know actually:) Also, cloud-related bits are used in the "mask2clouds" function (replaces the cloud pixels with zero), which is not used by default. I guess only cloud filtering is important, either masking or not should be fine.

Reply:
Ah, I see. So `--meta-cloud-name` is the flag that actually matters. I see Landsat uses `CLOUD_COVER`. Not sure how that's calculated, but I'm fine with trusting it to be useful.